Customizations for the xworld.mine.nu realm.

This is for using BusyBox in an initrd.

Index: sys-apps/busybox-1.11.1
===================================================================
--- sys-apps.orig/busybox-1.11.1
+++ sys-apps/busybox-1.11.1
@@ -1,7 +1,7 @@
 #
 # Automatically generated make config: don't edit
 # Busybox version: 1.11.1
-# Sat Sep 27 19:36:50 2008
+# Tue Sep 23 02:03:06 2008
 #
 CONFIG_HAVE_DOT_CONFIG=y
 
@@ -12,7 +12,7 @@ CONFIG_HAVE_DOT_CONFIG=y
 #
 # General Configuration
 #
-CONFIG_DESKTOP=y
+# CONFIG_DESKTOP is not set
 CONFIG_FEATURE_ASSUME_UNICODE=y
 CONFIG_FEATURE_BUFFERS_USE_MALLOC=y
 # CONFIG_FEATURE_BUFFERS_GO_ON_STACK is not set
@@ -30,7 +30,7 @@ CONFIG_FEATURE_SUID=y
 # CONFIG_FEATURE_SUID_CONFIG is not set
 # CONFIG_FEATURE_SUID_CONFIG_QUIET is not set
 # CONFIG_SELINUX is not set
-CONFIG_FEATURE_PREFER_APPLETS=y
+# CONFIG_FEATURE_PREFER_APPLETS is not set
 CONFIG_BUSYBOX_EXEC_PATH="/proc/self/exe"
 CONFIG_FEATURE_SYSLOG=y
 CONFIG_FEATURE_HAVE_RPC=y
@@ -56,12 +56,12 @@ CONFIG_WERROR=y
 CONFIG_NO_DEBUG_LIB=y
 # CONFIG_DMALLOC is not set
 # CONFIG_EFENCE is not set
-CONFIG_INCLUDE_SUSv2=y
+# CONFIG_INCLUDE_SUSv2 is not set
 
 #
 # Installation Options
 #
-CONFIG_INSTALL_NO_USR=y
+# CONFIG_INSTALL_NO_USR is not set
 CONFIG_INSTALL_APPLET_SYMLINKS=y
 # CONFIG_INSTALL_APPLET_HARDLINKS is not set
 # CONFIG_INSTALL_APPLET_SCRIPT_WRAPPERS is not set
@@ -76,12 +76,12 @@ CONFIG_PREFIX="./_install"
 #
 CONFIG_PASSWORD_MINLEN=6
 CONFIG_MD5_SIZE_VS_SPEED=2
-CONFIG_FEATURE_FAST_TOP=y
-CONFIG_FEATURE_ETC_NETWORKS=y
+# CONFIG_FEATURE_FAST_TOP is not set
+# CONFIG_FEATURE_ETC_NETWORKS is not set
 CONFIG_FEATURE_EDITING=y
 CONFIG_FEATURE_EDITING_MAX_LEN=1024
 CONFIG_FEATURE_EDITING_VI=y
-CONFIG_FEATURE_EDITING_HISTORY=15
+CONFIG_FEATURE_EDITING_HISTORY=200
 CONFIG_FEATURE_EDITING_SAVEHISTORY=y
 CONFIG_FEATURE_TAB_COMPLETION=y
 CONFIG_FEATURE_USERNAME_COMPLETION=y
@@ -90,7 +90,7 @@ CONFIG_FEATURE_VERBOSE_CP_MESSAGE=y
 CONFIG_FEATURE_COPYBUF_KB=4
 CONFIG_MONOTONIC_SYSCALL=y
 CONFIG_IOCTL_HEX2STR_ERROR=y
-CONFIG_FEATURE_HWIB=y
+# CONFIG_FEATURE_HWIB is not set
 
 #
 # Applets
@@ -103,11 +103,11 @@ CONFIG_AR=y
 CONFIG_FEATURE_AR_LONG_FILENAMES=y
 CONFIG_BUNZIP2=y
 CONFIG_BZIP2=y
-CONFIG_CPIO=y
-CONFIG_FEATURE_CPIO_O=y
+# CONFIG_CPIO is not set
+# CONFIG_FEATURE_CPIO_O is not set
 # CONFIG_DPKG is not set
-CONFIG_DPKG_DEB=y
-CONFIG_FEATURE_DPKG_DEB_EXTRACT_ONLY=y
+# CONFIG_DPKG_DEB is not set
+# CONFIG_FEATURE_DPKG_DEB_EXTRACT_ONLY is not set
 CONFIG_GUNZIP=y
 CONFIG_FEATURE_GUNZIP_UNCOMPRESS=y
 CONFIG_GZIP=y
@@ -120,10 +120,10 @@ CONFIG_FEATURE_TAR_GZIP=y
 CONFIG_FEATURE_TAR_BZIP2=y
 CONFIG_FEATURE_TAR_LZMA=y
 CONFIG_FEATURE_TAR_COMPRESS=y
-CONFIG_FEATURE_TAR_AUTODETECT=y
+# CONFIG_FEATURE_TAR_AUTODETECT is not set
 CONFIG_FEATURE_TAR_FROM=y
 CONFIG_FEATURE_TAR_OLDGNU_COMPATIBILITY=y
-CONFIG_FEATURE_TAR_OLDSUN_COMPATIBILITY=y
+# CONFIG_FEATURE_TAR_OLDSUN_COMPATIBILITY is not set
 CONFIG_FEATURE_TAR_GNU_EXTENSIONS=y
 CONFIG_FEATURE_TAR_LONG_OPTIONS=y
 CONFIG_FEATURE_TAR_UNAME_GNAME=y
@@ -135,13 +135,9 @@ CONFIG_UNZIP=y
 #
 # Common options for cpio and tar
 #
-
-#
-# Common options for dpkg and dpkg_deb
-#
-CONFIG_FEATURE_DEB_TAR_GZ=y
-CONFIG_FEATURE_DEB_TAR_BZ2=y
-CONFIG_FEATURE_DEB_TAR_LZMA=y
+# CONFIG_FEATURE_DEB_TAR_GZ is not set
+# CONFIG_FEATURE_DEB_TAR_BZ2 is not set
+# CONFIG_FEATURE_DEB_TAR_LZMA is not set
 
 #
 # Coreutils
@@ -179,7 +175,7 @@ CONFIG_FEATURE_EXPAND_LONG_OPTIONS=y
 CONFIG_EXPR=y
 CONFIG_EXPR_MATH_SUPPORT_64=y
 CONFIG_FALSE=y
-# CONFIG_FOLD is not set
+CONFIG_FOLD=y
 CONFIG_HEAD=y
 CONFIG_FEATURE_FANCY_HEAD=y
 CONFIG_HOSTID=y
@@ -188,7 +184,7 @@ CONFIG_INSTALL=y
 CONFIG_FEATURE_INSTALL_LONG_OPTIONS=y
 CONFIG_LENGTH=y
 CONFIG_LN=y
-# CONFIG_LOGNAME is not set
+CONFIG_LOGNAME=y
 CONFIG_LS=y
 CONFIG_FEATURE_LS_FILETYPES=y
 CONFIG_FEATURE_LS_FOLLOWLINKS=y
@@ -207,7 +203,7 @@ CONFIG_MV=y
 CONFIG_FEATURE_MV_LONG_OPTIONS=y
 CONFIG_NICE=y
 CONFIG_NOHUP=y
-# CONFIG_OD is not set
+CONFIG_OD=y
 CONFIG_PRINTENV=y
 CONFIG_PRINTF=y
 CONFIG_PWD=y
@@ -248,8 +244,8 @@ CONFIG_UNEXPAND=y
 CONFIG_FEATURE_UNEXPAND_LONG_OPTIONS=y
 CONFIG_UNIQ=y
 CONFIG_USLEEP=y
-# CONFIG_UUDECODE is not set
-# CONFIG_UUENCODE is not set
+CONFIG_UUDECODE=y
+CONFIG_UUENCODE=y
 CONFIG_WC=y
 CONFIG_FEATURE_WC_LARGE=y
 CONFIG_WHO=y
@@ -302,7 +298,7 @@ CONFIG_MKTEMP=y
 CONFIG_PIPE_PROGRESS=y
 CONFIG_RUN_PARTS=y
 CONFIG_FEATURE_RUN_PARTS_LONG_OPTIONS=y
-CONFIG_FEATURE_RUN_PARTS_FANCY=y
+# CONFIG_FEATURE_RUN_PARTS_FANCY is not set
 CONFIG_START_STOP_DAEMON=y
 CONFIG_FEATURE_START_STOP_DAEMON_FANCY=y
 CONFIG_FEATURE_START_STOP_DAEMON_LONG_OPTIONS=y
@@ -322,7 +318,7 @@ CONFIG_ED=y
 CONFIG_PATCH=y
 CONFIG_SED=y
 CONFIG_VI=y
-CONFIG_FEATURE_VI_MAX_LEN=4096
+CONFIG_FEATURE_VI_MAX_LEN=1024
 CONFIG_FEATURE_VI_8BIT=y
 CONFIG_FEATURE_VI_COLON=y
 CONFIG_FEATURE_VI_YANKMARK=y
@@ -398,22 +394,22 @@ CONFIG_ADDGROUP=y
 CONFIG_FEATURE_ADDUSER_TO_GROUP=y
 CONFIG_DELGROUP=y
 CONFIG_FEATURE_DEL_USER_FROM_GROUP=y
-CONFIG_FEATURE_CHECK_NAMES=y
+# CONFIG_FEATURE_CHECK_NAMES is not set
 CONFIG_ADDUSER=y
-CONFIG_FEATURE_ADDUSER_LONG_OPTIONS=y
+# CONFIG_FEATURE_ADDUSER_LONG_OPTIONS is not set
 CONFIG_DELUSER=y
 CONFIG_GETTY=y
 CONFIG_FEATURE_UTMP=y
 CONFIG_FEATURE_WTMP=y
 CONFIG_LOGIN=y
-CONFIG_PAM=y
+# CONFIG_PAM is not set
 CONFIG_LOGIN_SCRIPTS=y
-CONFIG_FEATURE_NOLOGIN=y
+# CONFIG_FEATURE_NOLOGIN is not set
 CONFIG_FEATURE_SECURETTY=y
 CONFIG_PASSWD=y
 CONFIG_FEATURE_PASSWD_WEAK_CHECK=y
-CONFIG_CRYPTPW=y
-CONFIG_CHPASSWD=y
+# CONFIG_CRYPTPW is not set
+# CONFIG_CHPASSWD is not set
 CONFIG_SU=y
 CONFIG_FEATURE_SU_SYSLOG=y
 CONFIG_FEATURE_SU_CHECKS_SHELLS=y
@@ -430,9 +426,9 @@ CONFIG_LSATTR=y
 #
 # Linux Module Utilities
 #
-CONFIG_DEPMOD=y
-CONFIG_FEATURE_DEPMOD_PRUNE_FANCY=y
-CONFIG_FEATURE_DEPMOD_ALIAS=y
+# CONFIG_DEPMOD is not set
+# CONFIG_FEATURE_DEPMOD_PRUNE_FANCY is not set
+# CONFIG_FEATURE_DEPMOD_ALIAS is not set
 CONFIG_INSMOD=y
 CONFIG_FEATURE_INSMOD_VERSION_CHECKING=y
 CONFIG_FEATURE_INSMOD_KSYMOOPS_SYMBOLS=y
@@ -486,8 +482,8 @@ CONFIG_MKFS_MINIX=y
 CONFIG_FEATURE_MINIX2=y
 CONFIG_GETOPT=y
 CONFIG_HEXDUMP=y
-CONFIG_FEATURE_HEXDUMP_REVERSE=y
-CONFIG_HD=y
+# CONFIG_FEATURE_HEXDUMP_REVERSE is not set
+# CONFIG_HD is not set
 CONFIG_HWCLOCK=y
 CONFIG_FEATURE_HWCLOCK_LONG_OPTIONS=y
 CONFIG_FEATURE_HWCLOCK_ADJTIME_FHS=y
@@ -499,7 +495,7 @@ CONFIG_FEATURE_MDEV_CONF=y
 CONFIG_FEATURE_MDEV_RENAME=y
 CONFIG_FEATURE_MDEV_RENAME_REGEXP=y
 CONFIG_FEATURE_MDEV_EXEC=y
-CONFIG_FEATURE_MDEV_LOAD_FIRMWARE=y
+# CONFIG_FEATURE_MDEV_LOAD_FIRMWARE is not set
 CONFIG_MKSWAP=y
 CONFIG_FEATURE_MKSWAP_V0=y
 CONFIG_MORE=y
@@ -508,19 +504,19 @@ CONFIG_VOLUMEID=y
 CONFIG_FEATURE_VOLUMEID_EXT=y
 CONFIG_FEATURE_VOLUMEID_REISERFS=y
 CONFIG_FEATURE_VOLUMEID_FAT=y
-CONFIG_FEATURE_VOLUMEID_HFS=y
-CONFIG_FEATURE_VOLUMEID_JFS=y
-CONFIG_FEATURE_VOLUMEID_XFS=y
+# CONFIG_FEATURE_VOLUMEID_HFS is not set
+# CONFIG_FEATURE_VOLUMEID_JFS is not set
+# CONFIG_FEATURE_VOLUMEID_XFS is not set
 CONFIG_FEATURE_VOLUMEID_NTFS=y
 CONFIG_FEATURE_VOLUMEID_ISO9660=y
-CONFIG_FEATURE_VOLUMEID_UDF=y
-CONFIG_FEATURE_VOLUMEID_LUKS=y
+# CONFIG_FEATURE_VOLUMEID_UDF is not set
+# CONFIG_FEATURE_VOLUMEID_LUKS is not set
 CONFIG_FEATURE_VOLUMEID_LINUXSWAP=y
-CONFIG_FEATURE_VOLUMEID_CRAMFS=y
-CONFIG_FEATURE_VOLUMEID_ROMFS=y
-CONFIG_FEATURE_VOLUMEID_SYSV=y
-CONFIG_FEATURE_VOLUMEID_OCFS2=y
-CONFIG_FEATURE_VOLUMEID_LINUXRAID=y
+# CONFIG_FEATURE_VOLUMEID_CRAMFS is not set
+# CONFIG_FEATURE_VOLUMEID_ROMFS is not set
+# CONFIG_FEATURE_VOLUMEID_SYSV is not set
+# CONFIG_FEATURE_VOLUMEID_OCFS2 is not set
+# CONFIG_FEATURE_VOLUMEID_LINUXRAID is not set
 CONFIG_MOUNT=y
 CONFIG_FEATURE_MOUNT_FAKE=y
 CONFIG_FEATURE_MOUNT_VERBOSE=y
@@ -533,11 +529,11 @@ CONFIG_FEATURE_MOUNT_FSTAB=y
 CONFIG_PIVOT_ROOT=y
 CONFIG_RDATE=y
 CONFIG_READPROFILE=y
-CONFIG_RTCWAKE=y
-CONFIG_SCRIPT=y
+# CONFIG_RTCWAKE is not set
+# CONFIG_SCRIPT is not set
 CONFIG_SETARCH=y
 CONFIG_SWAPONOFF=y
-CONFIG_FEATURE_SWAPON_PRI=y
+# CONFIG_FEATURE_SWAPON_PRI is not set
 CONFIG_SWITCH_ROOT=y
 CONFIG_UMOUNT=y
 CONFIG_FEATURE_UMOUNT_ALL=y
@@ -553,14 +549,14 @@ CONFIG_FEATURE_MTAB_SUPPORT=y
 #
 CONFIG_ADJTIMEX=y
 CONFIG_BBCONFIG=y
-CONFIG_CHAT=y
-CONFIG_FEATURE_CHAT_NOFAIL=y
-CONFIG_FEATURE_CHAT_TTY_HIFI=y
-CONFIG_FEATURE_CHAT_IMPLICIT_CR=y
-CONFIG_FEATURE_CHAT_SWALLOW_OPTS=y
-CONFIG_FEATURE_CHAT_SEND_ESCAPES=y
-CONFIG_FEATURE_CHAT_VAR_ABORT_LEN=y
-CONFIG_FEATURE_CHAT_CLR_ABORT=y
+# CONFIG_CHAT is not set
+# CONFIG_FEATURE_CHAT_NOFAIL is not set
+# CONFIG_FEATURE_CHAT_TTY_HIFI is not set
+# CONFIG_FEATURE_CHAT_IMPLICIT_CR is not set
+# CONFIG_FEATURE_CHAT_SWALLOW_OPTS is not set
+# CONFIG_FEATURE_CHAT_SEND_ESCAPES is not set
+# CONFIG_FEATURE_CHAT_VAR_ABORT_LEN is not set
+# CONFIG_FEATURE_CHAT_CLR_ABORT is not set
 CONFIG_CHRT=y
 CONFIG_CROND=y
 # CONFIG_DEBUG_CROND_OPTION is not set
@@ -575,8 +571,8 @@ CONFIG_FEATURE_DEVFS=y
 CONFIG_DEVMEM=y
 CONFIG_EJECT=y
 CONFIG_FEATURE_EJECT_SCSI=y
-CONFIG_FBSPLASH=y
-CONFIG_INOTIFYD=y
+# CONFIG_FBSPLASH is not set
+# CONFIG_INOTIFYD is not set
 CONFIG_LAST=y
 CONFIG_FEATURE_LAST_SMALL=y
 # CONFIG_FEATURE_LAST_FANCY is not set
@@ -597,7 +593,7 @@ CONFIG_FEATURE_HDPARM_HDIO_GETSET_DMA=y
 CONFIG_MAKEDEVS=y
 # CONFIG_FEATURE_MAKEDEVS_LEAF is not set
 CONFIG_FEATURE_MAKEDEVS_TABLE=y
-CONFIG_MAN=y
+# CONFIG_MAN is not set
 CONFIG_MICROCOM=y
 CONFIG_MOUNTPOINT=y
 CONFIG_MT=y
@@ -610,20 +606,20 @@ CONFIG_STRINGS=y
 # CONFIG_TASKSET is not set
 # CONFIG_FEATURE_TASKSET_FANCY is not set
 CONFIG_TIME=y
-CONFIG_TTYSIZE=y
+# CONFIG_TTYSIZE is not set
 CONFIG_WATCHDOG=y
 
 #
 # Networking Utilities
 #
-CONFIG_FEATURE_IPV6=y
-CONFIG_FEATURE_PREFER_IPV4_ADDRESS=y
+# CONFIG_FEATURE_IPV6 is not set
+# CONFIG_FEATURE_PREFER_IPV4_ADDRESS is not set
 CONFIG_VERBOSE_RESOLUTION_ERRORS=y
 CONFIG_ARP=y
 CONFIG_ARPING=y
-CONFIG_BRCTL=y
-CONFIG_FEATURE_BRCTL_FANCY=y
-CONFIG_FEATURE_BRCTL_SHOW=y
+# CONFIG_BRCTL is not set
+# CONFIG_FEATURE_BRCTL_FANCY is not set
+# CONFIG_FEATURE_BRCTL_SHOW is not set
 CONFIG_DNSD=y
 CONFIG_ETHER_WAKE=y
 CONFIG_FAKEIDENTD=y
@@ -632,8 +628,8 @@ CONFIG_FAKEIDENTD=y
 # CONFIG_FEATURE_FTPGETPUT_LONG_OPTIONS is not set
 CONFIG_HOSTNAME=y
 CONFIG_HTTPD=y
-CONFIG_FEATURE_HTTPD_RANGES=y
-CONFIG_FEATURE_HTTPD_USE_SENDFILE=y
+# CONFIG_FEATURE_HTTPD_RANGES is not set
+# CONFIG_FEATURE_HTTPD_USE_SENDFILE is not set
 CONFIG_FEATURE_HTTPD_RELOAD_CONFIG_SIGHUP=y
 CONFIG_FEATURE_HTTPD_SETUID=y
 CONFIG_FEATURE_HTTPD_BASIC_AUTH=y
@@ -643,24 +639,24 @@ CONFIG_FEATURE_HTTPD_CGI=y
 CONFIG_FEATURE_HTTPD_CONFIG_WITH_SCRIPT_INTERPR=y
 CONFIG_FEATURE_HTTPD_SET_REMOTE_PORT_TO_ENV=y
 CONFIG_FEATURE_HTTPD_ENCODE_URL_STR=y
-CONFIG_FEATURE_HTTPD_ERROR_PAGES=y
-CONFIG_FEATURE_HTTPD_PROXY=y
+# CONFIG_FEATURE_HTTPD_ERROR_PAGES is not set
+# CONFIG_FEATURE_HTTPD_PROXY is not set
 CONFIG_IFCONFIG=y
 CONFIG_FEATURE_IFCONFIG_STATUS=y
 CONFIG_FEATURE_IFCONFIG_SLIP=y
 CONFIG_FEATURE_IFCONFIG_MEMSTART_IOADDR_IRQ=y
 CONFIG_FEATURE_IFCONFIG_HW=y
 CONFIG_FEATURE_IFCONFIG_BROADCAST_PLUS=y
-CONFIG_IFENSLAVE=y
+# CONFIG_IFENSLAVE is not set
 CONFIG_IFUPDOWN=y
 CONFIG_IFUPDOWN_IFSTATE_PATH="/var/run/ifstate"
 CONFIG_FEATURE_IFUPDOWN_IP=y
 CONFIG_FEATURE_IFUPDOWN_IP_BUILTIN=y
 # CONFIG_FEATURE_IFUPDOWN_IFCONFIG_BUILTIN is not set
 CONFIG_FEATURE_IFUPDOWN_IPV4=y
-CONFIG_FEATURE_IFUPDOWN_IPV6=y
+# CONFIG_FEATURE_IFUPDOWN_IPV6 is not set
 CONFIG_FEATURE_IFUPDOWN_MAPPING=y
-CONFIG_FEATURE_IFUPDOWN_EXTERNAL_DHCP=y
+# CONFIG_FEATURE_IFUPDOWN_EXTERNAL_DHCP is not set
 # CONFIG_INETD is not set
 # CONFIG_FEATURE_INETD_SUPPORT_BUILTIN_ECHO is not set
 # CONFIG_FEATURE_INETD_SUPPORT_BUILTIN_DISCARD is not set
@@ -675,7 +671,7 @@ CONFIG_FEATURE_IP_ROUTE=y
 CONFIG_FEATURE_IP_TUNNEL=y
 CONFIG_FEATURE_IP_RULE=y
 CONFIG_FEATURE_IP_SHORT_FORMS=y
-CONFIG_FEATURE_IP_RARE_PROTOCOLS=y
+# CONFIG_FEATURE_IP_RARE_PROTOCOLS is not set
 CONFIG_IPADDR=y
 CONFIG_IPLINK=y
 CONFIG_IPROUTE=y
@@ -685,31 +681,31 @@ CONFIG_IPRULE=y
 # CONFIG_FEATURE_IPCALC_FANCY is not set
 # CONFIG_FEATURE_IPCALC_LONG_OPTIONS is not set
 CONFIG_NAMEIF=y
-CONFIG_FEATURE_NAMEIF_EXTENDED=y
+# CONFIG_FEATURE_NAMEIF_EXTENDED is not set
 CONFIG_NC=y
 CONFIG_NC_SERVER=y
 CONFIG_NC_EXTRA=y
 CONFIG_NETSTAT=y
-CONFIG_FEATURE_NETSTAT_WIDE=y
+# CONFIG_FEATURE_NETSTAT_WIDE is not set
 CONFIG_NSLOOKUP=y
 CONFIG_PING=y
-CONFIG_PING6=y
+# CONFIG_PING6 is not set
 CONFIG_FEATURE_FANCY_PING=y
-CONFIG_PSCAN=y
+# CONFIG_PSCAN is not set
 CONFIG_ROUTE=y
-CONFIG_SENDMAIL=y
-CONFIG_FETCHMAIL=y
-CONFIG_SLATTACH=y
+# CONFIG_SENDMAIL is not set
+# CONFIG_FETCHMAIL is not set
+# CONFIG_SLATTACH is not set
 CONFIG_TELNET=y
 CONFIG_FEATURE_TELNET_TTYPE=y
 CONFIG_FEATURE_TELNET_AUTOLOGIN=y
 CONFIG_TELNETD=y
 CONFIG_FEATURE_TELNETD_STANDALONE=y
 # CONFIG_TFTP is not set
-CONFIG_TFTPD=y
-CONFIG_FEATURE_TFTP_GET=y
-CONFIG_FEATURE_TFTP_PUT=y
-CONFIG_FEATURE_TFTP_BLOCKSIZE=y
+# CONFIG_TFTPD is not set
+# CONFIG_FEATURE_TFTP_GET is not set
+# CONFIG_FEATURE_TFTP_PUT is not set
+# CONFIG_FEATURE_TFTP_BLOCKSIZE is not set
 # CONFIG_DEBUG_TFTP is not set
 CONFIG_TRACEROUTE=y
 CONFIG_FEATURE_TRACEROUTE_VERBOSE=y
@@ -718,11 +714,11 @@ CONFIG_FEATURE_TRACEROUTE_USE_ICMP=y
 CONFIG_APP_UDHCPD=y
 CONFIG_APP_DHCPRELAY=y
 CONFIG_APP_DUMPLEASES=y
-CONFIG_FEATURE_UDHCPD_WRITE_LEASES_EARLY=y
+# CONFIG_FEATURE_UDHCPD_WRITE_LEASES_EARLY is not set
 CONFIG_DHCPD_LEASES_FILE="/var/lib/misc/udhcpd.leases"
 CONFIG_APP_UDHCPC=y
 CONFIG_FEATURE_UDHCPC_ARPING=y
-CONFIG_FEATURE_UDHCP_PORT=y
+# CONFIG_FEATURE_UDHCP_PORT is not set
 # CONFIG_FEATURE_UDHCP_DEBUG is not set
 CONFIG_FEATURE_RFC3397=y
 CONFIG_DHCPC_DEFAULT_SCRIPT="/usr/share/udhcpc/default.script"
@@ -733,8 +729,8 @@ CONFIG_FEATURE_WGET_STATUSBAR=y
 CONFIG_FEATURE_WGET_AUTHENTICATION=y
 CONFIG_FEATURE_WGET_LONG_OPTIONS=y
 CONFIG_ZCIP=y
-CONFIG_TCPSVD=y
-CONFIG_UDPSVD=y
+# CONFIG_TCPSVD is not set
+# CONFIG_UDPSVD is not set
 
 #
 # Process Utilities
@@ -745,22 +741,22 @@ CONFIG_KILL=y
 CONFIG_KILLALL=y
 CONFIG_KILLALL5=y
 CONFIG_NMETER=y
-CONFIG_PGREP=y
+# CONFIG_PGREP is not set
 CONFIG_PIDOF=y
 CONFIG_FEATURE_PIDOF_SINGLE=y
 CONFIG_FEATURE_PIDOF_OMIT=y
-CONFIG_PKILL=y
+# CONFIG_PKILL is not set
 CONFIG_PS=y
 CONFIG_FEATURE_PS_WIDE=y
-CONFIG_FEATURE_PS_TIME=y
-CONFIG_FEATURE_PS_UNUSUAL_SYSTEMS=y
+# CONFIG_FEATURE_PS_TIME is not set
+# CONFIG_FEATURE_PS_UNUSUAL_SYSTEMS is not set
 CONFIG_RENICE=y
 CONFIG_BB_SYSCTL=y
 CONFIG_TOP=y
 CONFIG_FEATURE_TOP_CPU_USAGE_PERCENTAGE=y
-CONFIG_FEATURE_TOP_CPU_GLOBAL_PERCENTS=y
-CONFIG_FEATURE_TOP_DECIMALS=y
-CONFIG_FEATURE_TOPMEM=y
+# CONFIG_FEATURE_TOP_CPU_GLOBAL_PERCENTS is not set
+# CONFIG_FEATURE_TOP_DECIMALS is not set
+# CONFIG_FEATURE_TOPMEM is not set
 CONFIG_UPTIME=y
 CONFIG_WATCH=y
 
@@ -806,9 +802,9 @@ CONFIG_ASH_EXPAND_PRMT=y
 # Bourne Shell Options
 #
 CONFIG_FEATURE_SH_EXTRA_QUIET=y
-CONFIG_FEATURE_SH_STANDALONE=y
-CONFIG_FEATURE_SH_NOFORK=y
-CONFIG_CTTYHACK=y
+# CONFIG_FEATURE_SH_STANDALONE is not set
+# CONFIG_FEATURE_SH_NOFORK is not set
+# CONFIG_CTTYHACK is not set
 
 #
 # System Logging Utilities
@@ -816,7 +812,7 @@ CONFIG_CTTYHACK=y
 CONFIG_SYSLOGD=y
 CONFIG_FEATURE_ROTATE_LOGFILE=y
 CONFIG_FEATURE_REMOTE_LOG=y
-CONFIG_FEATURE_SYSLOGD_DUP=y
+# CONFIG_FEATURE_SYSLOGD_DUP is not set
 CONFIG_FEATURE_IPC_SYSLOG=y
 CONFIG_FEATURE_IPC_SYSLOG_BUFFER_SIZE=16
 CONFIG_LOGREAD=y
@@ -855,6 +851,6 @@ CONFIG_SOFTLIMIT=y
 #
 # Print Utilities
 #
-CONFIG_LPD=y
-CONFIG_LPR=y
-CONFIG_LPQ=y
+# CONFIG_LPD is not set
+# CONFIG_LPR is not set
+# CONFIG_LPQ is not set
